submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submitter/RemoteExtSubmissionSubmitter.kt

```diff
@@ -79,6 +79,13 @@ class RemoteExtSubmissionSubmitter(
         TODO("Remote execution not required")
     }
 
+    override suspend fun indexToCleanRequest(
+        accNo: String,
+        version: Int,
+    ) {
+        TODO("Remote execution not required")
```

Comment:
Since this might be dealing with already existing submissions (i.e. the files might be in the new NFS), shouldn't this also be executed remotely?

Reply:
make sense yes